presto-hive-metastore/src/test/java/com/facebook/presto/hive/metastore/thrift/InMemoryHiveMetastore.java

```diff
@@ -172,15 +173,15 @@ public synchronized void alterDatabase(String databaseName, Database newDatabase
     public synchronized void createTable(Table table)
     {
         TableType tableType = TableType.valueOf(table.getTableType());
```

Comment:
Here along with other implementation at his level, org.apache.hadoop.hive.metastore.TableType is being called, and Hive 2.0 version doesn't have TableType.MATERIALIZED VIEW. So I'll move the table type casting logic to one level above, ThriftMetastoreUtil.fromMetastoreApiTable or toMetastoreApiTable.